_data/other.yml

```diff
@@ -28,6 +28,15 @@ websites:
       sms: Yes
       doc: https://support.airtable.com/hc/en-us/articles/219409917-Setting-up-two-factor-authentication
 
+    - name: Ancestry
+      url: https://www.ancestry.com
+      img: ancestry.png
+      tfa: Yes
+      sms: Yes
+      doc: https://support.ancestry.com/s/article/MFA
+      exceptions:
+          text: "2FA is currently only available for customers with an US based mobile phone number."
```

Comment:
```suggestion
          text: "2FA is currently only available for customers with a U.S. based phone number."
```

Reply:
@bdonvr As you have not allowed other users to commit to your branch, would you please add this code suggestion? 
Thanks! 
// Kai